packages/core/content-type-builder/server/src/controllers/validation/__tests__/content-type.test.ts

```diff
@@ -54,7 +54,7 @@ describe('Content type validator', () => {
             },
           },
         },
-      };
+      } as CreateContentTypeInput;
```

Comment:
CreateContentTypeInput doesn't handle the singularName, pluralName, etc within contentType (it's expecting it to be in info)